packages/microcosm/docs/api/microcosm.md

```diff
@@ -333,16 +333,18 @@ fork.push(getPeople)
 
 Create a new "group" action bound to the resolution of a list of
 actions. If all actions resolve or cancel, the group action will
-resolve. If any action is rejected, the group action fails:
++resolve. If any action is rejected, the group action fails. If all
+actions resolve, their respective payloads will be available in the
+`onDone` callback:
 
 ```javascript
 let group = repo.parallel([
   repo.push(actionOne),
   repo.push(actionTwo)
 ])
 
-group.onDone(function () {
-  console.log('hurrah!')
+group.onDone(function ([answer1, answer2]) {
+  console.log('hurrah!', answer1, answer2)
 })
```

Comment:
Long term, I think it would be great to also support objects, like:

```
let group = repo.parallel({
  one: repo.push(actionOne),
  two: repo.push(actionTwo)
})

let { one, two } = await group
```

I'll file an issue for it.

Reply:
https://github.com/vigetlabs/microcosm/issues/449